### PR TITLE
Support to ATtiny85 16Mhz internal PPL clock added

### PR DESCRIPTION
**Pull request by gioblu**
Marry Christmas :)
Would be confortable to have in the menu this option, very simple setup with 16Mhz clock generated by internal PPL clock see:
https://dntruong.wordpress.com/2015/07/08/setting-and-reading-attiny85-fuses/

### CODE REVIEW

File: attiny/boards.txt [hunk 1]
```diff
@@ -48,6 +48,16 @@ attiny85-8.build.f_cpu=8000000L
 attiny85-8.build.core=arduino:arduino
 attiny85-8.build.variant=tiny8
 
+attiny85-8.name=ATtiny85 (internal 16 MHz internal PPL clock)
```
Comment (sonyhome) [inline attiny/boards.txt]: I randomly stumbled on this useful code change to the AtTiny85 from my blog. :P

Was this tested? The flavor name must be changed too:
    attiny85-16.name

File: attiny/boards.txt [hunk 2]
```diff
@@ -48,6 +48,16 @@ attiny85-8.build.f_cpu=8000000L
...
+attiny85-8.bootloader.low_fuses=0xf1
+attiny85-8.bootloader.high_fuses=0xdd
```
Comment (sonyhome) [inline attiny/boards.txt]: To be consistent with other frequencies, the fuses should be: F1 DF FF
Though I contend that:
* Self programming is useful
* Brownout detection should be enabled
Hence changing the other frequency fuses to ?? DD FE would be better.

**Comment by damellis** [comment]
We definitely should add support for the 16 MHz PLL on the ATtiny25/45/85 but we should do it in the current master branch, so I'm closing this pull-request (which targets an old version of the boards.txt file).

See: https://github.com/damellis/attiny/issues/89

**Comment by gioblu** [comment]
Ciao @damellis, you want me to create a new issue on the actual master?
Thank you again and compliments for your great contribution to the arduino community.

**Comment by damellis** [comment]
That would be great! Note, that we should add the PLL clock only for the ATtiny25/45/85, not the ATtiny24/44/84, so you may need to distinguish them by creating two different items in the top-level boards menu. (Unless there's a way for the selection in one sub-menu, i.e. Processor, to affect the choices in another, i.e. Clock. But I don't think there is.)

**Comment by gioblu** [comment]
@damellis ok I am digging into it! Thank you for your suggestions.
You will receive a new pull request there soon.

**Comment by gioblu** [comment]
@damellis, ciao! You are right, there is no way to manage conditionally menu items.

So, a solution could be to have the menu structured this way:
attiny -> processor -> attiny85 -> 16 Mhz internal PPL clock
and remove the clock menu

This seems confortable for the user but adds overhead in the board file.
If you are ok with this solution I can send you the addon containing also the new menu structure.

**Comment by sonyhome** [comment]
Unless there's more logic control given by the menu declaration syntax,
I suspect the trick is to replace the menu hierarchy...

From:
|_ Processor
\_ Clock

To:
\_ Processor
    \_ Clock

So that the AtTiny85/45/25 get a different Clock menu naturally.

This might not bloat the boards.txt file too much...
It adds two clock menus, an nonPllClock and a pllClock.

Dan-

**Comment by gioblu** [comment]
ciao @sonyhome, you are right, I think this is the right solution, i will report your proposal in the related issue in the master version. Thank you for your suggestion.

**Comment by gioblu** [comment]
see https://github.com/damellis/attiny/issues/89 thank you again

**Pull request closed without merging**